packages/e2e-test-utils-playwright/src/admin/create-new-post.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import type { Admin } from './';
+
+interface NewPostOptions {
+	postType?: string;
+	title?: string;
+	content?: string;
+	excerpt?: string;
+	showWelcomeGuide?: boolean;
+}
+
+/**
+ * Creates new post.
+ *
+ * @param this
+ * @param options Options to create new post.
+ */
+export async function createNewPost(
+	this: Admin,
+	options: NewPostOptions = {}
+) {
+	const query = addQueryArgs( '', {
```

Comment:
Nit: could we just use `URLSearchParams` here? 😅

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/55099/commits/59720fe6b5072c37063c43887339ecd2653ca003 and https://github.com/WordPress/gutenberg/pull/55099/commits/d2aba51c6b6a3bedfcf5fabc23757a44328d472d